merlin/systems/triton/utils.py

```diff
@@ -223,11 +226,14 @@ def send_triton_request(
         triton_inputs = triton.convert_df_to_triton_input(schema, inputs, grpcclient.InferInput)
 
     outputs = [grpcclient.InferRequestedOutput(col) for col in outputs_list]
-    with client:
-        response = client.infer(triton_model, triton_inputs, request_id=request_id, outputs=outputs)
+
+    response = client.infer(triton_model, triton_inputs, request_id=request_id, outputs=outputs)
```

Comment:
removing this with is necessary to support calling this function mutliple times with the same client